Index: script.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- script.js	(date 1522564933000)
+++ script.js	(date 1522565544227)
@@ -14,4 +14,6 @@
 result = x+y;
 document.write(result);
 result = result + x;
+document.write("<br/>"+result);
+document.write("<br/>"+result);
 document.write("<br/>"+result);
\ No newline at end of file
